circle.yml

```diff
@@ -341,7 +341,7 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - stage
       # Deploy production
       - frontend-build:
           name: frontend-build-production
```

Comment:
@gabor-boros I just noticed that the `frontend-build-production` step still requires `frontend-deploy-stage`, which I don't think is appropriate now that stage and production branches are separate. I think it should require `frontend-test` instead.

Reply:
My initial idea was to have at least one deploy of the change on stage to be able to deploy on production, but that would cause more confusion, so I just changed this in 1f74190 for the behaviour you just wrote.